examples/qdrantdb/preFilters.ts

```diff
@@ -4,12 +4,21 @@ import {
   Document,
   MetadataMode,
   QdrantVectorStore,
+  Settings,
   VectorStoreIndex,
-  runWithCallbackManager,
-  serviceContextFromDefaults,
   storageContextFromDefaults,
 } from "llamaindex";
 
+// Update callback manager
+Settings.callbackManager = new CallbackManager({
+  onRetrieve: (data) => {
+    console.log(
+      "The retrieved nodes are:",
+      data.nodes.map((node) => node.node.getContent(MetadataMode.NONE)),
+    );
+  },
+});
+
```

Comment:
@himself65 just saw that you removed `runWithCallbackManager` here ;)
How about his simplification:
```suggestion
Settings.callbacks = {
  onRetrieve: (data) => {
    console.log(
      "The retrieved nodes are:",
      data.nodes.map((node) => node.node.getContent(MetadataMode.NONE)),
    );
  },
};

```